ModelBundle/Document/ContentType.php

```diff
@@ -211,7 +211,15 @@ public function setDefaultListable(array $defaultListable)
      */
     public function getName($language = 'en')
     {
-        return $this->names->get($language)->getValue();
+        if ($this->names->containsKey($language)) {
+            return $this->names->get($language)->getValue();
+        }
+
+        if ($this->names->count() > 0) {
+            return $this->names->first()->getValue();
```

Comment:
I think this function should only return the asked language, and not a random one if the asked one is not defined.
Maybe the function should be modified to accept 2 parameters :
 - the first one, mandatory, specifying the asked language,
 - the second one, listing fallback languages as an array.